devops/windows/base-docker-wsl/a-images-1-pull-from-hub.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+set -u
+
+source ./conf.env
+source ./conf/images/conf-hub-images.sh
+
+docker pull $REGISTRY_IMAGE
+#docker pull $KEYCLOAK_IMAGE
+docker pull $HAPROXY_IMAGE
+#docker pull $NGINX_IMAGE
```

Comment:
Is there any reason we are keeping these commented out code lines if you are not going to use them?

Reply:
NGINX can come out 
KEYCLOCK must stay 
   for standalone reference with KEYCLOAK we need a KEYCLOAK server --- this is the image